elastalert/elastalert.py

```diff
@@ -624,7 +625,8 @@ def run_query(self, rule, start=None, end=None, scroll=False):
             pass
 
         if 'scroll_id' in rule:
-            rule.pop('scroll_id')
+            scroll_id = rule.pop('scroll_id')
+            self.current_es.clear_scroll(scroll_id=scroll_id)
```

Comment:
@Qmando For clearing the es scroll, we have to send the scroll id in the body, i applied below change and my scroll is maintained at the max limit i set.
self.thread_data.current_es.clear_scroll(body={'scroll_id': scroll_id})